### PR TITLE
Add olm templates for the cdi-operator

### PR DESCRIPTION
**Pull request by rthallisey**
Integrate cdi-operator with OLM

### CODE REVIEW

File: cdi-operator-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+apiVersion: v1
+metadata:
+  name: cdi-operator
+  namespace: openshift-operator-lifecycle-manager
+data:
+  customResourceDefinitions: |-
+    - apiVersion: apiextensions.k8s.io/v1beta1
+      kind: CustomResourceDefinition
+      metadata:
+        name: cdis.cache.example.com
```
Comment (irosenzw) [inline cdi-operator-configmap.yaml]: change to cdis.cdi.kubevirt.io

File: cdi-operator-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+      spec:
+        version: v1alpha1
+        group: cache.example.com
```
Comment (irosenzw) [inline cdi-operator-configmap.yaml]: change to cdi.kubevirt.io

File: cdi-operator-configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+          mediatype: image/png
+
+        install:
+          strategy: deployment
+          spec:
+            permissions:
+            - serviceAccountName: cdi-operator
+              rules:
+              - apiGroups:
+                - cache.example.com
```
Comment (irosenzw) [inline cdi-operator-configmap.yaml]: same here

File: cdi-operator-configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+                          fieldRef:
+                            fieldPath: metadata.namespace
+                      - name: OPERATOR_NAME
+                        value: "cdi-operator"
+        maturity: stable
+        version: 1.3.0
+
+        customresourcedefinitions:
+          owned:
+          - name: cdis.cache.example.com
```
Comment (irosenzw) [inline cdi-operator-configmap.yaml]: same here

**Comment by rthallisey** [comment]
closing in favor of https://github.com/kubevirt/cdi-operator/pull/11

**Pull request closed without merging**